pheps/phep-9999.md

```diff
@@ -0,0 +1,171 @@
+   - Software License: indicates level of software license compliance with the [PyHC Standards Document](https://doi.org/10.5281/zenodo.2529131) 
+   - DOI: indicates whether or not a package's software repository has a DOI (e.g., from Zenodo) 
+   - pip installable?: indicates whether a package is pip installable
+   - conda installable?: indicates whether a package is conda installable
+
+The following table shows the benefits that are associated with each tier:  
+
+|       | **Gold** | **Silver** | **Bronze** | **Copper** | 
+| :-: | :----------: | :-----------: | :------------: | :-------------: |
+| Summer School Inclusion \* | Yes | Yes** | No | No |
+| PyHC-top-tier Env Inclusion \* | Yes | Yes ** | No | No | 
+| PyHC-all Env Inclusion \* | Yes | Yes | Yes | No |
+| PyHC-Chat Bot Inclusion \* | Yes | Yes ** | No | No |
+| pyOpenSci Verified Badge | Yes | No | No | No |
+| Standards Compliance Assistance \* | Yes *** | Yes | Yes | Yes |
+| Main Page Listing | Yes | Yes | Yes | No |
+| Secondary Page Listing | No | No | No | Yes |
+| HSSI Inclusion | Yes | Yes | Yes | Yes | 
+| Conference Travel Funding | Yes | Yes | Yes | No |  
+
+\*  = can opt-in or opt-out of this  
+
+\** = conditional upon justification on importance/use of package (e.g. number of users, level of commitment within PyHC activities, or effort made for those items but gold level not yet achieved)  
+
+\*** = conditional upon justification that package is in danger of dropping down to a Silver tier 
+
+Descriptions for each heading are as follows:
+   - Summer School Inclusion: indicates whether a package will be included in summer school teaching materials
+   - PyHC-top-tier Env Inclusion: indicates whether a package will be included within the current PyHC software environment used at the summer school  
+   - PyHC-all Env Inclusion: indicates whether a package will be included within the current PyHC software environment containing all packages Bronze and higher.
+   - PyHC-Chat Bot Inclusion: indicates whether a packages will have up-to-date information included within the ChatGPT4-powered PyHC-Chat bot
+   - pyOpenSci Verified Badge: a badge that shows whether a package has completed the pyOpenSci review process
+   - Standards Compliance Assistance: indicates whether a package will receive extra help and/or funding (dependent on available PyHC resources) for conforming to the PyHC standards
+   - Main Page Listing: indictes whether a package's information will be displayed on a new, main PyHC Project page
+   - Secondary Page Listing: indicates whetheer a package's information will be displayed on a new, secondary PyHC Project page 
+   - HSSI Inclusion: indicates whether a package will be included within the Heliophysics Software Search Interface
+   - Conference Travel Funding: indicates whether developers from a package will be considered for travel funding assistance to relevant science conferences (e.g. SHINE, CEDAR, or GEM)
```

Comment:
Do we really have control over this? Can it be "Community recommended for conference travel funding?"

Reply:
Well, yes and yes. Personally, I would not want a lower level package even informally representing PyHC at a science conference. Preferably, those would be silver and gold.

Reply:
@jtniehof yes. Basically because of what Rebecca said. I also wrote into the next PyHC leadership grant some funding to send package leads to conferences, so I want some kind of guidelines for how to pick said packages. I think it makes sense to have packages that have put in the work to develop in line with PyHC needs to be the ones representing us at meetings.

Reply:
Got it. Could this be "considered for PyHC travel funding assistance..."? Someone can put in their own TWSC proposal without this...

Reply:
Ah @jtniehof I see now where the confusion came in. I'll add in PyHC so it's clear what travel funding money pot we're referring to.